### PR TITLE
First version operator-sdk 1.0.0

### PR DESCRIPTION
**Pull request by vboulineau**

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -35,60 +33,35 @@ unit_tests:
...
   image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/docker-notary:0.6.1
+  before_script: []
   script:
     - IMAGE_NAME=$DOCKER_REGISTRY/$PROJECTNAME:$TARGET_TAG
     - RELEASE_IMAGE_NAME=$DOCKER_REGISTRY/$PROJECTNAME:$CI_COMMIT_TAG
     - docker pull $IMAGE_NAME
     - docker tag $IMAGE_NAME $RELEASE_IMAGE_NAME
     - docker push $RELEASE_IMAGE_NAME
 
-test_e2e:
```
Comment (ahmed-mez) [inline .gitlab-ci.yml:73]: nit: let's rename the `unit_tests` job since it runs the new e2e tests from now on

**Review by ahmed-mez — APPROVED** [review]
amazing work 👏  (few nits and questions)
the changes are significant, would be also nice to have extra reviews from @DataDog/container-integrations

File: RELEASING.md [hunk 2]
```diff
@@ -3,12 +3,9 @@
...
 
 1. Checkout the repository on the correct branch and changeset (`master`).
-2. Run the pre-release script:
-
-    - For a release candidate, run the following command, locally: `make VERSION=x.v.z RELEASE_CANDIDATE=1 pre-release`
-
-      For example, to generate the release version `0.3.0-rc.2`, run: `make VERSION=0.3.0 RELEASE_CANDIDATE=2 pre-release`
-    - For a final release, run the following command, locally: `make VERSION=x.v.z pre-release`
+2. Run the bundle generation:
+    - For a release candidate, run the following command, locally: `make VERSION=x.v.z bundle`
```
[the remark this answers is not in the document]
Comment (vboulineau) [reply RELEASING.md:7]: Yes, correct

File: .github/workflows/validation.yml [hunk 3]
```diff
@@ -23,25 +23,10 @@ jobs:
         make install-tools
     - name: run build
       run: |
-        make build
-    - name: run unit-test
+        make manager
+    - name: run unit tests and E2E tests (fake cluster)
       run: |
         make test
-    - name: run validate
```
Comment (clamoriniere) [inline .github/workflows/validation.yml:30]: do we have an equivalent command? is it included in `make manager` ?

File: Makefile [hunk 4]
```diff
@@ -1,132 +1,191 @@
...
-install-tools: bin/golangci-lint bin/operator-sdk bin/yq
+bin/openapi-gen:
+	go build -o ./bin/openapi-gen k8s.io/kube-openapi/cmd/openapi-gen
 
 bin/yq:
-	hack/install-yq.sh
+	./hack/install-yq.sh 3.3.0
 
 bin/golangci-lint:
 	hack/golangci-lint.sh v1.18.0
```
Comment (clamoriniere) [inline Makefile:185]: to be consistent with other make target.

```suggestion
	./hack/golangci-lint.sh v1.18.0
```

File: README.md [hunk 5]
```diff
@@ -15,21 +15,21 @@
...
 
 First, deploy the Custom Resources Definitions:
 
 ```console
-$ kubectl apply -f ./deploy/crds
+$ make install
 ```
 
-If you want to deploy a specific controller version, the version can be set in the `deploy/operator.yaml` file. By default, the `latest` docker image tag is used.
+Then deploy default manifest (uses Kustomize)
```
Comment (clamoriniere) [inline README.md:26]: should we add Kustomize cli as project dependency? are we using only kustomize included in the `kustomize apply -k` option?

File: .github/workflows/validation.yml (hunk 3 above)
Comment (vboulineau) [reply .github/workflows/validation.yml:30]: Yes `manager` depends on `lint`

File: .gitlab-ci.yml (hunk 1 above)
Comment (vboulineau) [reply .gitlab-ci.yml:73]: Done

File: README.md (hunk 5 above)
Comment (vboulineau) [reply README.md:26]: Yes, the generated Makefile deals with that automatically

**Review by clamoriniere — APPROVED** [review]

**Pull request merged**